bin/example_component.jsx.txt

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import "./COMPONENT_NAME.less";
```

Comment:
Do we ever expect to produce stateless components? In that case the function syntax is cleaner, but this does seem like it would be the common case; maybe leave a comment to say that?

Also, should we be leaving doc comments on the classes? Especially if my roommate Prakash's component explorer supports extracting those.